common/rakelib/utils.rake

```diff
@@ -0,0 +1,74 @@
+# Used constants:
+# none
+
+class Utils
+  # formatter types
+  :xcpretty   # pass through xcpretty and store in artifacts
+  :raw        # store in artifacts
+  :to_string  # run using backticks and return output
+
+  # run a command using xcrun and xcpretty if applicable
+  def self.run(cmd, task, subtask = '', xcrun: false, formatter: :raw)
+    commands = xcrun ? [*cmd].map { |cmd|
+      "#{version_select} xcrun #{cmd}"
+    } : [*cmd]
+
+    case formatter
+    when :xcpretty then xcpretty(commands, task, subtask)
+    when :raw then plain(commands, task, subtask)
+    when :to_string then `#{commands.join(' && ')}`
+    else raise "Unknown formatter '#{formatter}'"
+    end
+  end
+
+  # print an info header
+  def self.print_info(str)
```

Comment:
Right, so:
- rename current `self.print_info` to `print_header`
- add a print_info
- add a print_error